CONTRIBUTING.md

```diff
@@ -6,6 +6,35 @@ Before submitting new features or changes to current functionality, it is recomm
 [open an issue](https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/new)
 and discuss your ideas or propose the changes you wish to make.
 
+
+## Breaking Changes
+
+When your PR introduces a breaking change:
+
+* Add the `breaking change` label to your PR
```

Comment:
most contributors can't add labels to their PRs

Reply:
i've added some comment automation for this